it/controllers/TaxYearsControllerISpec.scala

```diff
@@ -61,10 +62,11 @@ class TaxYearsControllerISpec extends ComponentSpecBase with FeatureSwitching {
           verifyIncomeSourceDetailsCall(testMtditid)
 
           Then("The view should have the correct headings and all tax years display")
+          val expectedYearsShown = getCurrentTaxYearEnd.getYear - propertyAccountingStartLocalDate.getYear + 1
```

Comment:
I think changing propertyAccountingStartLocalDate value to **currentTaxYear - 5** will be better. Also other fields in the PropertyDetailsIntegrationTestConstants file needs to be changed accordingly. But if its taking more time then will leave it.

Reply:
updated